File: app/src/main/java/com/neome/api/meta/base/dto/StudioFieldGeoPoint.java
// neome.ai API - do not change
//

package com.neome.api.meta.base.dto;

import com.neome.api.meta.base.Types.GeoPoint;
import com.neome.api.meta.base.Types.MetaIdField;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class StudioFieldGeoPoint extends StudioFieldEditable
{
  @Nullable
  public MetaIdField defaultFieldId;

  @Nullable
  public GeoPoint defaultValue;
}

File: app/src/main/java/com/neome/api/meta/base/dto/StudioFieldGeoPoint.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.Types.GeoPoint
import com.neome.api.meta.base.Types.MetaIdField

open class StudioFieldGeoPoint : StudioFieldEditable() {
    var defaultFieldId: MetaIdField? = null
    var defaultValue: GeoPoint? = null
}
